client/cypress/integration/editor-view_spec.ts

```diff
@@ -63,6 +63,15 @@ describe('Editor View', () => {
       });
     });
 
+    it('should fill the available viewport height', () => {
+      editorView.getEditorLayoutMain().then(main => {
+        const main = main[0];
```

Comment:
It's not really good practice to give this the same name as the parameter. So I think we should either rename this one, use array destructuring or maybe use `async/await`? If `async/await` works with Cypress, that would definitely be the preferred way, together with array destructuring.

```ts
const [main] = await editorView.getEditorLayoutMain();
```

Reply:
I'm gonna try that!